tests/security/with-api-prefix/account.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
```

Comment:
Rename`with-api-prefix` to `api` to match path.

Reply:
done.